src/admin/components/elements/ReactSelect/index.tsx

```diff
@@ -24,6 +63,41 @@ const ReactSelect: React.FC<Props> = (props) => {
     showError && 'react-select--error',
```

Comment:
I'd like to see another className added conditionally for sortable that has `cursor: hand` to help users identify the dragable feature. It would also help developers add their own css styles for customizing the look for sortable items.

Reply:
Implemented.